### DISCUSSION TITLE
MIP-4: Deep link navigation

### DISCUSSION THREAD

**Discussion by hmalik88** [MetaMask/metamask-improvement-proposals#52] [MIPs]
Please discuss here [MIP-4](https://github.com/MetaMask/metamask-improvement-proposals/pull/51) here.

**Comment by vandan** [comment]
Just adding some references for similar dapp-to-wallet deepLinking patterns being used (mainly for in-app browsers in Mobile Wallet apps):
WalletConnect - https://eips.ethereum.org/EIPS/eip-1328
Phantom [deeplinks](https://docs.phantom.app/phantom-deeplinks/deeplinks-ios-and-android)
Coinbase SDK [deeplink](https://docs.cdp.coinbase.com/wallet-sdk/docs/deep-link) 
MetaMask SDK also has some [deeplinking](https://docs.metamask.io/wallet/concepts/sdk-connections/) features

I haven't seen any general purpose standards around these patterns though.
